### ISSUE TITLE
Suggested fixes for "showcase view whole nearby activity"

### ISSUE THREAD

**Issue by amire80** [commons-app/apps-android-commons#2872]
**Summary:** 

A couple of things should be fixed for the benefit of the users and the translators.

**Steps to reproduce:** 

Check the messages "showcase view whole nearby activity" and "search this area".

The message "showcase view whole nearby activity" currently says: "Clicking on 'SEARCH THIS AREA' locks the map and launches a nearby search around that location."

The words "SEARCH THIS AREA" in the message above probably refer to the message with the key "search this area", the text of which is "Search this area" - not in all-caps. Several things to fix in the messages' text:
* The capitalization must be consistent.
* Don't use all-caps for any kind of emphasis or design that must be translatable because most alphabets don't have lowercase and uppercase distinction.

Furthermore, perhaps the message "showcase view whole nearby activity" can automatically include the text of the other message, so that the consistency would be enforced and translators wouldn't have to copy things.

**Comment by fredrikjonasson** [comment]
I would like to start my contribution with this one.

**Comment by fredrikjonasson** [comment]
![Screenshot_1555621111](https://user-images.githubusercontent.com/17785337/56391412-dc90b700-622e-11e9-926f-1d5d274a1830.png)
![Screenshot_1555621192](https://user-images.githubusercontent.com/17785337/56391413-dc90b700-622e-11e9-8451-cfbb715142d2.png)

I took a quick look and it seems to be consistent capitalization towards the user. The key "Search this area" indeed seems to refer to the text as you describe. Should it be changed anyway to be consistent towards the developer?

**Comment by amire80** [comment]
Yes. The capitalization should be consistent. My suggestion is not to use ALL CAPS, but to write "Search this area", because there are many languages in which ALL CAPS are irrelevant, such as Chinese, Hindi, and Arabic. _However_, maybe the designer of the app has a different opinion about this. The important thing is that it will be consistent.

And if you can automatically insert "Search this area" into "Clicking on 'SEARCH THIS AREA'" as a parameter, then the capitalization will become consistent automatically. (By "parameter" I mean something like `%1$s`.)

**Comment by neslihanturan** [comment]
Hi @HarunJr , thanks for your interest but currently related classes are being edited. So what about waiting until #2969 is merged and after that if the issue still persist you can solve.

**Comment by HarunJr** [comment]
No problem @neslihanturan. I shall wait.

**Comment by FawziyahAlebiosu** [comment]
Looks to me like inside of the strings.xml file, search this area has been removed and is no longer capitalized. See here:
https://github.com/commons-app/apps-android-commons/blob/0ad269d3d6ef461c74d6338368eb81e4cdf5200c/app/src/main/res/values/strings.xml#L452
Can this issue be closed now?

**Comment by FawziyahAlebiosu** [comment]
I spoke too soon!
@HarunJr looks like all that is left is for this line to be made to lowercase and then both strings will be consistent:
https://github.com/commons-app/apps-android-commons/blob/0ad269d3d6ef461c74d6338368eb81e4cdf5200c/app/src/main/res/values/strings.xml#L310

**Comment by macgills** [comment]
Buttons in android by default capitalise all the text that you set to them so I doubt `SEARCH THIS AREA` was ever in the codebase as all caps. I am not 100% on what to do here because referencing our button `SEARCH THIS AREA` with the non capitalised text `Search this Area` weakens their connection in English.

I'd say the best way it to probably paramterise `showcase_view_whole_nearby_activity` and Uppercase `search_this_area` as works by locale.

@neslihanturan any opinions on how this should look?

**Comment by Henry4669** [comment]
Hi, is this issue resolved? If not, then can I contribute to this issue?

**Comment by Henry4669** [comment]
@neslihanturan Hi, sorry I forgot to ping one of the contributors. 
If I can contribute to this issue, may I ask which activities the strings are used in?

**Comment by Henry4669** [comment]
Hi @nicolas-raoul, sorry for bothering you. @neslihanturan doesn't seem to be active but I have done exactly as @amire80 had asked, which involved, for example, changing the capitalisation properties of a button in the map activity.
![image](https://user-images.githubusercontent.com/115204041/198504624-8045a225-c5de-4da2-a9d3-426d1f840da6.png)
I would like your opinion on how the button should look. Do you think it should be changed?
Also, please assign me if you think it needs to be changed.

**Comment by devbytespark** [comment]
Hey @nicolas-raoul could I work on this seems like a simple fix I can test it on my Pixel 6

**Comment by nicolas-raoul** [comment]
Indeed, the button has been removed recently I believe.
